Allow for zero frequency sound objects (#2806)
OpenLoco/OpenLoco: CHANGELOG.md

- Fix: [#2778] News has celebration sound for player company going bankrupt.
- Fix: [#2785] Unique names for industries incorrectly generated.
- Fix: [#2791] Station cargo image incorrect for high quantities of cargo.

> - Fix: [#2806] Objects with zeroed base sound frequency do not play.

Reviewer: ```suggestion
- Fix: [#2806] Sound objects with zeroed base frequency are inaudible.
```

Author: Its actually a vehicle object

Reviewer: All the more reason to clarify it, then, I think.
